src/patternfly/demos/DataToolbar/examples/DataToolbar.md

```diff
@@ -0,0 +1,524 @@
+---
+title: Data Toolbar
+section: demos
+---
+
+import './DataToolbar.css'
+
+## Demos
+```hbs title=Data-toolbar-attribute-value-search-filter-desktop
+{{#> data-toolbar data-toolbar--id="data-toolbar-attribute-value-search-filter-desktop"}}
+  {{#> data-toolbar-content}}
+    {{#> data-toolbar-group data-toolbar-group--modifier="pf-m-toggle-group pf-m-show-on-lg"}}
```

Comment:
You can use `pf-m-show` here without a breakpoint to always show (since it's a desktop example).